bindata/networking-console-plugin/001-config-map.yaml

```diff
@@ -9,6 +9,7 @@ data:
       keepalive_timeout  65;
       server {
         listen              9443 ssl;
+        listen              [::]:9443 ssl;
```

Comment:
I believe this will break on clusters have have IPv6 disabled. You probably need a fix like https://github.com/openshift/console-plugin-template/pull/79

Reply:
It looks like this is an unsupported configuration. Disregard